src/vellum/workflows/runner/runner.py

```diff
@@ -73,9 +72,7 @@ def __init__(
         parent_context: Optional[ParentContext] = None,
     ):
         if state and external_inputs:
-            raise ValueError(
-                "Can only run a Workflow providing one of state or external inputs, not both"
-            )
+            raise ValueError("Can only run a Workflow providing one of state or external inputs, not both")
```

Comment:
@marlenc3 do you have any special width configured locally? I'm going to add black and flake to our precommits later tn so we don't keep ping ponging haha. But i think our default is 120

Reply:
I do not but I can add in the default 120! 🫡